### ISSUE TITLE
Scan function for string

### ISSUE THREAD

**Issue by Jae0504** [intel/qpl#34]
Hi, I have a question about qpl scan function. I can only check that there is an example of utilizing scan function to find a 1byte size character. However, is there any way to find a string with a scan function? such as "hello" in the source file.

**Comment by mzhukova** [comment]
Hi @Jae0504,
scan operation could also be used for search in a certain range, please see [this documentation page](https://intel.github.io/qpl/documentation/dev_guide_docs/c_use_cases/filter_ops/c_operations_op_scan.html). Please, let me know if that's what you were looking for.

**Comment by Jae0504** [comment]
Hi @mzhukova ,
Thank you for your reply!
I checked that there is a scan range option. In my understanding, I think we can only set the range from one ASCII character to another ASCII character. For example, 'a' to 'z.'
If I set the range from 'h' to 'o,' then I can get hello from the source file. However, I will also get a single character from 'h' to 'o' which is not relevant to the string that I want. Is there anyway to get only specific string with scan function?
